RecoTracker/TkHitPairs/src/HitPairGeneratorFromLayerPair.cc

```diff
@@ -109,6 +109,29 @@ HitDoublets HitPairGeneratorFromLayerPair::doublets( const TrackingRegion& regio
 
 }
 
+
+HitDoublets HitPairGeneratorFromLayerPair::doublets( const TrackingRegion& region,
+						    const edm::Event & iEvent, const edm::EventSetup& iSetup, const Layer& innerLayer, const Layer& outerLayer) {
```

Comment:
It would be possible to reduce copy-paste by making the overload above to call this overload (but again I'm fine with a later cleanup PR).